nvflare/app_opt/pt/params_converter.py

```diff
@@ -12,15 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Dict
+from typing import Dict, List
```

Comment:
This change has nothing to do with ScriptExecutor. Please use a different PR.

Reply:
Since ScriptExecutor is no longer an Executor itself and cannot register the TensorDecomposer, we need this for the pytorch version of ScriptExecutor to work